saleor/product/models.py

```diff
@@ -81,35 +84,37 @@ class ProductClass(models.Model):
     has_variants = models.BooleanField(
         pgettext_lazy('Product class field', 'has variants'), default=True)
     product_attributes = models.ManyToManyField(
-        'ProductAttribute', related_name='products_class', blank=True,
+        'ProductAttribute',
+        related_name='products_class',
+        blank=True,
         verbose_name=pgettext_lazy('Product class field',
                                    'product attributes'))
     variant_attributes = models.ManyToManyField(
-        'ProductAttribute', related_name='product_variants_class', blank=True,
-        verbose_name=pgettext_lazy(
-            'Product class field', 'variant attributes'))
+        'ProductAttribute',
+        related_name='product_variants_class',
+        blank=True,
+        verbose_name=pgettext_lazy('Product class field',
+                                   'variant attributes'))
     is_shipping_required = models.BooleanField(
         pgettext_lazy('Product class field', 'is shipping required'),
         default=False)
 
     class Meta:
-        verbose_name = pgettext_lazy(
-            'Product class model', 'product class')
-        verbose_name_plural = pgettext_lazy(
-            'Product class model', 'product classes')
+        verbose_name = pgettext_lazy('Product class model', 'product class')
+        verbose_name_plural = pgettext_lazy('Product class model',
+                                            'product classes')
```

Comment:
The original formatting was fine - we usually try to break the line in the first possible place, in this case after the first parenthesis.

Reply:
Ok, I did a little too much autoformatting here in editor. Seams I need some changes in yapf config.